cookbook/controller/service.rst

```diff
@@ -277,8 +277,7 @@ controller:
         in the :class:`Symfony\\Component\\Routing\\Generator\\UrlGeneratorInterface`.
 
 :method:`Symfony\\Bundle\\FrameworkBundle\\Controller\\Controller::getDoctrine` (service: ``doctrine``)
-
-    *Simply inject doctrine instead of fetching it from the container*
+    Simply inject doctrine instead of fetching it from the container.
```

Comment:
Yes, we can maintain the italic text. I removed it because it looked wrong to me. I've just restored it.